### PR TITLE
T7319 fixes for bisection

### PR DESCRIPTION
**Pull request by gctucker**
This is a series of fixes mostly useful for adding an initial automated bisection job.

It replaces #29.

### CODE REVIEW

File: lava-v2-jobs-from-api.py [hunk 1]
```diff
@@ -136,6 +141,9 @@ def main(args):
                     dtb_full = dtb
                     if arch == 'arm64':
                         dtb = str(dtb).split('/')[-1]
+                    dtb_target = dtb.partition('.dtb')[0]
+                    if targets and dtb_target not in targets:
```
Comment (mattface) [inline lava-v2-jobs-from-api.py]: This seems to be switching the --targets arg to use dtb names instead of LAVA device type names. Admittedly only the cambridge lab used to have this set, and we don't use it any more, but why the change?

Comment (gctucker) [reply lava-v2-jobs-from-api.py]: The `--targets` option is used for bisections to only run boot tests on one device type.

It's true that this change looks wrong though, I did this a month ago to fix some issue but the bisection code now works without it so I'll drop it and update the PR.  Well spotted!

**Comment by gctucker** [comment]
@mattface so I see what the problem is now, the backend is triggering bisections with the dtb name rather than the LAVA device type.  For example, I'm getting some jobs with the target set to `am335x-boneblack`.  This looks like something that needs to be fixed in the backend, will take a look.

**Comment by gctucker** [comment]
@mattface FYI pull request for the fix in kernelci-backend:
https://github.com/kernelci/kernelci-backend/pull/36

**Comment by gctucker** [comment]
@mattface the kernelci-backend changes have now been merged so progress can resume on this one (was to trigger bisections with the device type name rather than dtb name).

**Comment by gctucker** [comment]
@mattface Please let me know if you still want to review this and #31 or if it's OK to merge them on staging now and I'll do it.
